### DISCUSSION TITLE
Welcome to DrawBoard-Online Discussions!

### DISCUSSION THREAD

**Discussion by forrestyuan** [forrestyuan/DrawBoard-Online#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yisizheng** [comment]
目前只能进行共享，不能进行多人协作吧，多个人同时在画的就会出现混乱
